### PR TITLE
fix saving 1D meshes in vtk and vtu formats

### PR DESCRIPTION
**Pull request by vlukes**

### CODE REVIEW

File: sfepy/discrete/fem/meshio.py [hunk 1]
```diff
@@ -403,6 +403,11 @@ def write(self, filename, mesh, out=None, ts=None, **kwargs):
         if self.file_format in ['vtk', 'vtu']:
             ngkey = 'node_groups'
             cgkey = 'mat_id'
+            if coors.shape[1] != 3:
+                nnd, ndim = coors.shape
+                ndim = 3 - ndim
+                coors = nm.hstack([coors,
```
Comment (rc) [inline sfepy/discrete/fem/meshio.py:409]: meshio uses `numpy.pad()` - might be faster?

**Comment by rc** [comment]
Thanks! LGTM, only one question regarding the padding function, see above.

File: sfepy/discrete/fem/meshio.py (hunk 1 above)
Comment (vlukes) [reply sfepy/discrete/fem/meshio.py:409]: I hope that this way is fast enough. From my point of view, this solution is more readable than   using `numpy.pad()`.

Comment (rc) [reply sfepy/discrete/fem/meshio.py:409]: OK

**Pull request merged**